src/shared/abbreviators.ts

```diff
@@ -45,16 +45,44 @@ const ABBREVIATION_LENGTHS = [
   [30, 0],
 ];
 
+const ABBREVIATION_LENGTHS_SOLANA = [
```

Comment:
Probably should rename `ABBREVIATION_LENGTHS` to `ABBREVIATION_LENGTHS_EVM`.

Reply:
Addressed in https://github.com/rolod0x/rolod0x/pull/490/commits/d5db96d1a9a550dc39d8c31a82100ca86634e708